lms/djangoapps/teacher_dashboard/urls.py

```diff
@@ -3,19 +3,30 @@
 """
 from django.conf.urls import patterns, url
 
+
+API_PREFIX = r'teacher/api/v0'
+
+
 urlpatterns = patterns(
-        '',
-        url(r'^teacher_dashboard/?$', 'teacher_dashboard.views.dashboard_view', name='dashboard_view_handler'),
+    '',
+    url(r'^teacher_dashboard/?$', 'teacher_dashboard.views.dashboard_view', name='dashboard_view_handler'),
 
-        # Api calls
-        url(r'^licenses/?$', 'teacher_dashboard.views.licenses_api_call'),
-        url(r'^licenses/(?P<license_pk>[a-zA-Z0-9\-]+)/simulations/?$', 'teacher_dashboard.views.simulations_api_call'),
-        url(
-            r'^licenses/(?P<license_pk>[a-zA-Z0-9\-]+)/simulations/(?P<simulation_pk>[a-zA-Z0-9]+)/students/?$',
-            'teacher_dashboard.views.students_api_call'
+    # Api calls
+    url(r'^{}/licenses/?$'.format(API_PREFIX), 'teacher_dashboard.views.licenses_api_call'),
```

Comment:
How do think about little refactoring?

```
teacher_db_patterns = pattern(
    'teacher_dashboard', 
    url(r'^/licenses/?$', 'licenses_api_call')
    ...
)

urlpatterns = patterns(
    ...
     url(r'^/teacher/api/v0/?$', include(teacher_db_patterns),
)
```
Like in https://docs.djangoproject.com/en/1.9/topics/http/urls/#including-other-urlconfs and  https://docs.djangoproject.com/en/1.9/ref/urls/#patterns